content/pl_docker/docker_course_dev.md

```diff
@@ -173,3 +173,49 @@ When developing locally on PrairieLearn, you **must** click "Load from Disk" eve
 
 That's it!
 You're ready to develop questions for the OPB!
+
+# Setting up local autograder container
```

Comment:
This should be a Level Two heading (there can only be one level heading on a markdown page). I suggest `##` here, and then increase the indent level for subsequent sections and subsections by one - so `##` would go to `###`

Reply:
Resolved.